src/routes/github/github-oauth-router.ts

```diff
@@ -208,5 +209,5 @@ const getCloudOrGHESAppClientSecret = async (gitHubAppConfig, jiraHost: string)
 // IMPORTANT: We need to keep the login/callback/middleware functions
 // in the same file as they reference each other
 export const GithubOAuthRouter = Router();
-GithubOAuthRouter.get("/login", GithubOAuthLoginGet);
-GithubOAuthRouter.get(callbackSubPath, GithubOAuthCallbackGet);
+GithubOAuthRouter.get("/login", jiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubOAuthLoginGet);
```

Comment:
Remove `jiraSymmetricJwtMiddleware` and `GithubServerAppMiddleware`?

Reply:
the old double dipping middleware, good catch!

Reply:
cancel that, this is what we want... for now.

the duplication of the router make it confusing, but this will be above the .use for the two middlewares in question